src/OpenTracing/Tracer.php

```diff
@@ -14,6 +14,39 @@
 interface Tracer
 {
     /**
+     * @deprecated use either startActiveSpan or startManualSpan instead.
```

Comment:
We are not yet 1.0, so I would just go and remove this instead of deprecating.

Reply:
It makes sense to me @felixfbecker?

Reply:
Nice. I just changed it.